Neos.Flow/Classes/Http/Request.php

```diff
@@ -95,6 +97,11 @@ class Request extends BaseRequest implements ServerRequestInterface
      */
     const ATTRIBUTE_BASE_URI = 'baseUri';
 
+    /**
+     * PSR-7 Attribute containing the Flow unified arguments (get, post, files) for this request.
+     */
+    const ATTRIBUTE_UNIFIED_ARGUMENTS = 'unifiedArguments';
```

Comment:
The unified arguments are a core feature of Flows HTTP Stack and we need to make them somehow available. I don't even know, if there's any other way to access the POST params in PSR-7 Request.

Reply:
See below, I made it a helper that is used "as needed" to avoid having all the arguments laying around two times.